tsconfig.json

```diff
@@ -3,8 +3,10 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "moduleResolution": "node",
+    "resolveJsonModule": true,
     "strict": true,
     "skipLibCheck": true,
+    "target": "ES2021",
```

Comment:
Stops a type error using `String.prototype.replaceAll()` in code that runs in Node.